lib/katello/permissions/ui_permissions.rb

```diff
@@ -0,0 +1,5 @@
+require 'katello/plugin'
+
+Foreman::Plugin.find(:katello).security_block :katello_ui do
+  permission :view_katello_ui, 'bastion/bastion' => [:index, :index_ie]
```

Comment:
I don't think they should, as a user, this would be a strange permission to encounter I believe and the individual routes are covered where needed.